app/views/sidebar/_dashboard.html.erb

```diff
@@ -7,12 +7,31 @@
         <span class="sr-only">Toggle Dropdown</span>
       </a>
       <div class="dropdown-menu dropdown-menu-right mt-2">
-        <a class="dropdown-item" href="/post?n=15322&title=How%20to%20...&tags=activity:new,draft"><%= translation('dashboard._header.dropdown.share_activity') %></a>
-        <a class="dropdown-item" href="/post"><%= translation('dashboard._header.dropdown.share_research_note') %></a>
-        <a class="dropdown-item" href="/questions/new?tags=question:general"><%= translation('dashboard._header.dropdown.ask_question') %></a>
-        <a class="dropdown-item" href="/post?template=event&tags=event"><%= translation('dashboard._header.dropdown.post_event') %></a>
-        <div class="dropdown-divider"></div>
-        <a class="dropdown-item" href="/post?tags=blog-submission"><%= translation('dashboard._header.dropdown.tell_story_blog') %></a>
+        <% if current_user&.has_tag('translation-helper') && I18n.locale != :en %>
```

Comment:
Ooh, actually... this is interesting. This repetition does make for a lot more code... i wonder... what do you think, is there any way to reduce repetition in this while still achieving what we want? What are some ideas to try to brainstorm this? 

Sorry to pause a moment, but I hope my hesitation makes sense, @imajit -- still very much appreciate this work!

Reply:
Yeah, I agree there is a repetition in the code as I'm wrapping the button inside a div so that the globe icon stays inline with the text. 
-  If I keep the `if` part only, then the button size reduces for all users
- If I keep the `else` part, then the globe icon overflows to the next line
- I can't store it in a new file as partial as the repetition  is kind of nested inside each `div`
So, I did this bit ugly implementation to satisfy all cases.  One way to avoid this is if I reduce the number of globe icons in the navbar, in that case, there will be less code repetition and fewer number of globe icons on the Dashboard. 
For now, I can't see a different way of implementation though :confused:

Reply:
Hmm.... I totally appreciate this is a tough one. Thanks for thinking hard on it. What if... we did it in JavaScript, after rendering? Could that work?

Reply:
I added a helper function to render the dropdown button, this reduces some repetition